geant-incubator/Issues/edugain-proxy/description.xml

```diff
@@ -0,0 +1,113 @@
+<IssueSpecification version="4">
+	<Name>
+		<en>Demo GEANT Incubator eduGAIN proxy</en>
+        <nl>Demo GEANT Incubator eduGAIN proxy</nl>
```

Comment:
GÉANT is spelled sometimes as `GÉANT`, sometimes as `GEANT`. Is there a reason for that? Otherwise, ideally this is made consistent.

Additionally, here and at various other places in this PR tabs are mixed with spaces.

Reply:
Agreed